python/lsst/jointcal/jointcal.py

```diff
@@ -614,6 +609,67 @@ def _do_load_refcat_and_fit(self, associations, defaultFilter, center, radius,
 
         return result
 
+    def _load_reference_catalog(self, refObjLoader, center, radius, filterName,
+                                applyColorterms=False):
```

Comment:
Could this method be moved out of `jointcal` and into somewhere else (`pipe_tasks`, I guess?) where it could be a stand-alone method?  I think it could be generally useful (I'd be writing something almost exactly like it for `fgcmcal`), and the only reference to `self` seems to be for the logging.